readme.md

```diff
@@ -49,6 +49,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [iOS](https://github.com/vsouza/awesome-ios)
 - [Android](https://github.com/JStumpp/awesome-android)
 - [IoT & Hybrid Apps](https://github.com/weblancaster/awesome-IoT-hybrid)
+- [Tensorflow](https://github.com/jtoy/awesome-tensorflow/)
```

Comment:
Should also be added to the end of the section. Please actually read https://github.com/sindresorhus/awesome/blob/master/contributing.md ...